sz/Traffic/src/traffic/relogo/UserObserver.groovy

```diff
@@ -16,11 +16,11 @@ class UserObserver extends ReLogoObserver{
 		clearAll()
 		setDefaultShape(UserTurtle, "car")
 		
-		def w = (UserPatch.laneWidth + 1) / 2
+		def width = (UserPatch.laneWidth + 1) / 2
 		def d = 15
```

Comment:
What does the `d` do here? 🙂

Reply:
This is the distance from origin to the location of each spawn point.

Reply:
👍